srcpkgs/Ice/template

```diff
@@ -1,86 +1,60 @@
 # Template file for 'Ice'
 pkgname=Ice
-version=3.5.1
-revision=22
-build_wrksrc=cpp
-makedepends="zlib-devel bzip2-devel mcpp-devel db-devel expat-devel libressl-devel"
+version=3.7.5
+revision=1
+wrksrc="ice-${version}"
+build_style=gnu-makefile
+make_build_args="LANGUAGES=cpp prefix=${DESTDIR}/usr OPTIMIZE=yes
+ USR_DIR_INSTALL=yes CONFIGS=shared CONFIGS+=cpp11-shared"
+make_install_args="LANGUAGES=cpp prefix=${DESTDIR}/usr"
+makedepends="zlib-devel bzip2-devel zeroc-mcpp db-devel expat-devel
+ libressl-devel lmdb-devel editline-devel"
 short_desc="Internet Communications Engine (Ice)"
-maintainer="Orphaned <orphan@voidlinux.org>"
-license="GPL-2.0-only, ICE"
-homepage="http://www.zeroc.com"
-distfiles="${homepage}/download/Ice/${version%.*}/${pkgname}-${version}.tar.gz"
-checksum=989e51194c6adadbd156da3288e37bad847b93b3b876502e83033b70493af392
-python_version=2 #unverified
+maintainer="a dinosaur <nick@a-dinosaur.com>"
+license="GPL-2.0-only, custom:ICE"
+homepage="https://www.zeroc.com"
+distfiles="https://github.com/zeroc-ice/ice/archive/v${version}.tar.gz"
+checksum=36bf45591a95e6ee7216153d45d8eca05ff00c1da35608f0c400e6ddc8049da9
+patch_args="-Np1"
 
 if [ "$CROSS_BUILD" ]; then
-	# Cross build requires the host's slice2cpp and slice2freeze
-	hostmakedepends+=" Ice"
+	# Cross build requires the host's slice2cpp
+	hostmakedepends+=" ${pkgname}"
 fi
```

Comment:
I prefer to use `Ice` here :p 

pkgname is unlikely to change, so this doesn't save much work.

Reply:
Done.